### PR TITLE
Moving workshop examples from TRAIN-101 to XC-101

### PR DESCRIPTION
**Pull request by mfsilva22**
This PR also fixes a broken link for the standard assessment set

### CODE REVIEW

File: exampleCourse/questions/workshop/Lesson4_example1/question.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<pl-question-panel>
+<p>
+<!-- Give an example of a non-zero $2\times 2$-matrix $A=\begin{bmatrix} a_{11} & a_{12} \\ a_{21} & a_{22} \end{bmatrix}$ such that $A^2$ is the zero matrix. -->
```
Comment (nicknytko) [inline exampleCourse/questions/workshop/Lesson4_example1/question.html:3]:
```suggestion
```

File: exampleCourse/questions/workshop/Lesson4_example3/server.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+                    data["format_errors"][name] = 'Temperature {} looks too low.'.format(name)
+            elif name in ["D", "L", "x1"]:
+                if (var > 0.5) :
+                    data["format_errors"][name] = 'Dimension {} is outside the range of acceptable values for this experiment. Check your units?'.format(name)
+
+def grade(data):
+
+    Ac = math.pi*data['submitted_answers']['D']**2/4
+    qf = -data['params']['k']*Ac*(data['submitted_answers']['T1'] - data['submitted_answers']['Tb'])/data['submitted_answers']['x1']
+    #data['correct_answers']['qf'] = qf
```
Comment (nicknytko) [inline exampleCourse/questions/workshop/Lesson4_example3/server.py:34]:
```suggestion
```

**Comment by nicknytko** [comment]
image

Maybe this is a lot of work, but I'm thinking that we should prefix these workshop questions with `Workshop: `, similarly to how the demo questions have `Demo: ` in front of them.  We could even give them a topic of "Workshop Demo" so people can sort and find them on the questions page.

**Comment by mfsilva22** [comment]
> Please change

Okay, I will make the changes soon. Thanks

**Comment by nicknytko** [comment]
> > Please change
> 
> Okay, I will make the changes soon. Thanks

Haha, no worries, I just wanted to make this more clear in the PR page :slightly_smiling_face:

**Comment by mwest1066** [comment]
Could we move the video links on each day's page to the top? At the moment they are at the bottom of each page.

**Comment by mwest1066** [comment]
@nicknytko I just talked to Mariana and she's crushed with work for the next week. I'd love to get this deployed so the Maryland and Michigan State folks can do the tutorial (they were asking for this today). Would you be able to make the updates that you suggested? 😄

**Comment by nicknytko** [comment]
> @nicknytko I just talked to Mariana and she's crushed with work for the next week. I'd love to get this deployed so the Maryland and Michigan State folks can do the tutorial (they were asking for this today). Would you be able to make the updates that you suggested? smile

:+1:

**Comment by mwest1066** [comment]
Mariana says that we also need to update the docs page to point the Workshop Day N questions links at XC 101 questions rather than TRAIN 101 questions. And they need to point to questions in the course, not the course instance.

**Pull request merged**